packages/perspective-viewer-d3fc/src/js/series/seriesRange.js

```diff
@@ -10,9 +10,17 @@ import * as d3 from "d3";
 import {domain} from "../axis/mainAxis";
 
 export function seriesLinearRange(settings, data, valueName) {
-    return d3.scaleLinear().domain(domain().valueName(valueName)(data));
+    return d3.scaleLinear().domain(
+        domain()
+            .valueName(valueName)(data)

```

Comment:
Do we need to call data twice?
Would the following work?
`domain()
            .valueName(valueName)
            .pad([0, 0])(data)`

Reply:
Oh, I think that's a copying mistake, good spot